aten/src/TH/generic/THTensorEvenMoreMath.cpp

```diff
@@ -65,6 +65,8 @@ void THTensor_(nonzero)(THLongTensor *subscript, THTensor *tensor)
                 );
   delete [] sizes;
   delete [] idx;
+
+#undef IS_NONZERO
 }
 
 accreal THTensor_(sumall)(THTensor *tensor)
```

Comment:
this means we are exposing sumall on half, right?

Reply:
I can remove it, but I don't think this exposes it to python (for any tensor type). I was leaving it just for simplicity since it was implemented for Bool as well, the ifndef's can get ugly with too many. As far as I can tell it's just used by other c++ functions in this file, not in declarations.cwrap or native_functions.yaml.

Reply:
oh, I don't think it effects python, but it either:
1) (if the definition exists) increases the binary size and exposes code that isn't tested (so someone may come along later and start using it without realizing it isn't actually tested)
2) (if the definition doesn't exist) can cause errors at link or import time if someone actually tries to call it.

Reply:
I think I can actually just remove this for bool too

Reply:
removed for half (and kept bool, it was used somewhere)